pkg/ring/ring.go

```diff
@@ -34,6 +34,9 @@ const (
 
 	// CompactorRingKey is the key under which we store the compactors ring in the KVStore.
 	CompactorRingKey = "compactor"
+
+	// StoreGatewayRingKey is the key under which we store the store gateways ring in the KVStore.
+	StoreGatewayRingKey = "store-gateway"
```

Comment:
None of these keys belong here, but that's a separate PR.

Reply:
Agree. Will take care in a separate PR.